packages/nc-gui/components/dlg/AirtableImport.vue

```diff
@@ -403,22 +403,30 @@ function downloadLogs(filename: string) {
             </a-checkbox>
           </div>
 
-          <!--          Import Users Columns -->
+          <!--          Import Formula Columns -->
           <div class="my-2">
-            <a-checkbox v-model:checked="syncSource.details.options.syncUsers">
-              {{ $t('labels.importUsers') }}
-            </a-checkbox>
+            <a-tooltip placement="top">
+              <template #title>
+                <span>{{ $t('title.comingSoon') }}</span>
+              </template>
+              <a-checkbox v-model:checked="syncSource.details.options.syncFormula" disabled>
+                {{ $t('labels.importFormulaColumns') }}
+              </a-checkbox>
+            </a-tooltip>
           </div>
 
-          <!--          Import Formula Columns -->
-          <a-tooltip placement="top">
-            <template #title>
-              <span>{{ $t('title.comingSoon') }}</span>
-            </template>
-            <a-checkbox v-model:checked="syncSource.details.options.syncFormula" disabled>
-              {{ $t('labels.importFormulaColumns') }}
-            </a-checkbox>
-          </a-tooltip>
+          <!--          Invite Users 
+          <div class="my-2">
+            <a-tooltip placement="top">
+              <template #title>
+                <span>{{ $t('title.comingSoon') }}</span>
+              </template>
+              <a-checkbox v-model:checked="syncSource.details.options.syncUsers" disabled>
+                {{ $t('labels.importUsers') }}
+              </a-checkbox>
+            </a-tooltip>
+          </div>
+          -->
```

Comment:
Can be removed if no longer needed or going to enable in future.

Reply:
We've disabled for now as it is not functioning properly, there is a possibility to re-enable